velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -995,5 +996,298 @@ TEST_F(StringTest, trim) {
       trimWithTrimStr("\u6570", "\u6574\u6570 \u6570\u636E!"),
       "\u6574\u6570 \u6570\u636E!");
 }
+
+TEST_F(StringTest, split) {
+  auto limit = -1;
+  auto delim = ",";
+  auto numRows = 3;
+  auto data = std::vector<std::string>{
+      {"I,he,she,they"}, // Simple
+      {"one,,,four,"}, // Empty strings
+      {""}, // The whole string is empty
+  };
+
+  auto createStrings = [&](RowSet /*rows*/) {
+    return makeFlatVector<StringView>(
+        numRows, [&](vector_size_t row) { return StringView{data[row]}; });
```

Comment:
Why do we create vectors with the same elements? I assume we don't need to test duplicate values.

Reply:
not same right ? `data[row]` @rui-mo

Reply:
I do refactor and put the test in seperated test file `SplitTest.cpp`